### DISCUSSION TITLE
ECE%20Meng%28coop%29%40Waterloo/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#149] [Announcements]
# ECE%20Meng%28coop%29%40Waterloo/

https://csmsapp.org/ECE%20Meng%28coop%29%40Waterloo/

**Comment by yuhengcai1** [comment]
这项目最多A+吧，这能到S？

**Reply by yuhengcai1** [reply]
甚至觉得最多是A

**Reply by xichenpan** [reply]
just init commit. 看datapoints今年还没录的，拒了若干其他学校录的还不错的。后面出了更多结果可以再根据调整

**Reply by lezhangleonard** [reply]
A-差不多，毕竟是个弱校ece项目

**Reply by PkuCuipy** [reply]
选校梯度以针对陆本的**录取难度**作为排序标准，和弱校强校何干

**Reply by HUST-AI-HYZ** [reply]
难录取 ≠ 项目质量高

**Reply by Capital-Hu** [reply]
对10043学校同学来说这是S了 :(
我目前还没看到25fall录取的10043学生

**Comment by Skylar-xty** [comment]
感觉UofT ECE Meng是不是今年（25fall）能同级了，也开始有coop了
